mappings/net/minecraft/entity/passive/PassiveEntity.mapping

```diff
@@ -1,15 +1,22 @@
 CLASS ags net/minecraft/entity/passive/PassiveEntity
 	FIELD a breedingAge I
 	FIELD bD CHILD Lpr;
+	FIELD bE entityWidth F
+	FIELD bF entityHeight F
```

Comment:
What about `lastWidth/Height`, because these values are set by `setSize(FF)` and are read by `setSize(F)` which sets the size to `lastWidth/Height` (multiplied by `multiplier`)?